v20.2/performance-benchmarking-with-tpc-c-100k-warehouses.md

```diff
@@ -346,57 +269,42 @@ Before running the benchmark, it's important to allocate partitions to workload
 
     {% include copy-clipboard.html %}
     ~~~ shell
-    $ ulimit -n 200500 && ./workload run tpcc \
-    --partitions 27 \
-    --warehouses 100000 \
-    --partition-affinity 0,5,10,15,20,25 \
-    --ramp 30m \
-    --duration 1ms \
-    $(cat addrs)
+    ulimit -n 500500 && ./workload run tpcc --partitions 81 \
+    --warehouses 150000 --active-warehouses $NUMWARE \
```

Comment:
Asked in the gdoc but probably better here: what is the value of `$NUMWARE`?  I assume it's a var used when scripting.  Is it the same as `--warehouses` or something different?